server/src/controllers/requestController.js

```diff
@@ -18,14 +14,27 @@ export default class requestController {
    * @returns {Array} request
    */
   static getAll(req, res) {
+    let filter = '';
+    let presentQuery = '';
+    if (req.query.category) {
+      req.query.category = req.query.category.toLowerCase();
+      req.query.category = validator.trim(req.query.category);
+      filter = `WHERE category LIKE '${req.query.category}%'`;
+      presentQuery = 'in this category';
+    }
     if (req.query.status) {
       req.query.status = req.query.status.toLowerCase();
       req.query.status = validator.trim(req.query.status);
+      filter = `WHERE status LIKE '${req.query.status}%'`;
+      presentQuery = 'of this status';
+    }
+    if (req.query.status || req.query.category) {
       pool.connect()
```

Comment:
what is pool ?

Reply:
I am using connection pool to make query. And it is required that I connect to a client through the pool.connect() method

Reply:
Same as above ^